### PR TITLE
Compression demo test

### PR DESCRIPTION
**Pull request by shailesh33**
Add a basic compression test in the demo app with hmset/hmget in a pipeline.

### CODE REVIEW

File: dyno-demo/src/main/java/com/netflix/dyno/demo/redis/DynoJedisDemo.java [hunk 1]
```diff
@@ -922,6 +954,10 @@ public static void main(String args[]) throws IOException {
                 demo.runSScanTest(true);
                 break;
             }
+            case 9: {
+                    demo.runCompressionTest();
```
Comment (ipapapa) [inline dyno-demo/src/main/java/com/netflix/dyno/demo/redis/DynoJedisDemo.java]: `runCompressionInPipeline`?

**Comment by ipapapa** [comment]
Do we also need a unit test for compression and pipelines or is this enough?

**Comment by shailesh33** [comment]
This is not really a unit test. It does not run as part of the build. This is a demo app that we can run locally as part of dev test and is subject to change. I am committing the code just for reference if we need it next time.

**Comment by shailesh33** [comment]
I have renamed the test function

**Pull request merged**